README.rst

```diff
@@ -56,11 +56,13 @@ Links
 
 * Feedback and ideas: http://mail.python.org/mailman/listinfo/code-quality
 
-* Cyclomatic complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity.
+* Cyclomatic complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity
 
 * Ned Batchelder's script:
   http://nedbatchelder.com/blog/200803/python_code_complexity_microtool.html
 
+* McCabe complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity
```

Comment:
That ended up there, because I updated your branch to match the rest of the README. This is exactly where I want it. =)